modules/end-user-guide/partials/proc_creating-a-workspace-from-workspace-sample.adoc

```diff
@@ -0,0 +1,30 @@
+// Module included in the following assemblies:
+//
+// creating-a-workspace-from-included-options
+
+[id="creating-a-workspace-from-workspace-sample_{context}"]
+= Creating a workspace from workspace sample
+
+You can create a new workspace by using one of the workspace samples included in {prod-short}.
+
+.Prerequisites
+
+* A running instance of {prod-short}
```

Comment:
.

Reply:
Period omitted deliberately. IBM Style Guide: p. 89 "_Write lists so that all or none of the items start with complete sentences. ... If list items comprise only sentence fragments, do not include any end punctuation_."